karma.conf.js

```diff
@@ -6,6 +6,7 @@ module.exports = function(config) {
     files : [
       'http://ajax.googleapis.com/ajax/libs/angularjs/1.4.9/angular.js',
       'ts_output_readonly_do_NOT_change_manually/src/gameLogic.js',
+      'ts_output_readonly_do_NOT_change_manually/src/common.js',
```

Comment:
Need to update here if you add another ts files so that `jasmine` know where to find it.